joomla/joomla-cms
# Add Sortable Function to Repeatable

Pull Request for New Function
#### Summary of Changes

I have change the JFormFieldRepeatable i add DropDown Sortablefunction like the ordering in Articles
and fix the repeatable.js for displaying images preview after select an image in the Repeatable Modalbox
#### Testing Instructions
1. Install the Test Modul from here [https://github.com/nielsnuebel/mod_testrepeatable/releases/tag/1.0.1](https://github.com/nielsnuebel/mod_testrepeatable/releases/tag/1.0.1) Download the Zip File.
2. After you have install the Modul go to Extensions->Moduls and create a new Modul from Type "Test Sortable Repeatable Field"
3. Add a Title and a Modul Position like position-7
4. Modul Assignment "On all pages"
5. Go to the Tab "Module" and add two Items - click on the Button "Select" a Modal Box will popup
6. First item Title "Title 1" and image choose what you want, check first if the Preview Box will shown the image
7. Second item "Title 2" and a image Click Save in the Modalbox and 
8. Click Save to save the changes
9. Go to the Frontend and check the result, there is a Unsortlist with a headline and a image
10. go to the Module in the Backend and click the Button Select and put the Second item over the first item, for that click on the three lines on the left side and hold the mouse down and drag it over the first row.
11. Save the repeatable changes and the module change and check the result on the frontpage
